src/Feliz.Plotly/Feliz.Plotly.fsproj

```diff
@@ -5,6 +5,18 @@
     <PackageId>Feliz.Plotly.V2</PackageId>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Version>2.28.0</Version>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Authors>Kurt Mueller</Authors>
+    <PackageTags>fsharp fable react visualization graph chart</PackageTags>
+    <PackageDescription>Fable bindings written in the Feliz-style for plotly.js</PackageDescription>
+    <!-- you can fill in this property with whatever release notes you want users to see on
+    NuGet.Org -->
+    <PackageReleaseNotes></PackageReleaseNotes>
+    <!-- We don't care about the size of the .NET dll, since user apps are going to use the 
+      compiled JS, so just embed the PDB in the DLL so users can go-to-definition right to the source code -->
+    <DebugType>embedded</DebugType>
+   <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
Adding these lines makes users able to f12 in Ionide or VS and go to the actual F# sources during development.